parsimonious/tests/test_nodes.py

```diff
@@ -145,6 +156,33 @@ def visit_greeting(self, thing, visited_children):
 
 
 def test_node_inequality():
-    node = Node('text', 'o hai', 0, 5)
+    node = Node(Literal('12345'), 'o hai', 0, 5)
     ok_(node != 5)
     ok_(node != None)
+    ok_(node != Node(Literal('23456'), 'o hai', 0, 5))
+    ok_(not (node != Node(Literal('12345'), 'o hai', 0, 5)))
+
+
+def test_generic_visit_NotImplementedError_unnamed_node():
+    class MyVisitor(NodeVisitor):
+        grammar = Grammar(r'''
+            bar = "b" "a" "r"
+        ''')
+        unwrapped_exceptions = (NotImplementedError, )
+
+    with assert_raises(NotImplementedError) as e:
+        MyVisitor().parse('bar')
+    assert_in('No visitor method was defined for "b"', str(e.exception))
+
+
+def test_generic_visit_NotImplementedError_named_node():
+    class MyVisitor(NodeVisitor):
+        grammar = Grammar(r'''
+            bar = myrule myrule myrule
+            myrule = ~"[bar]"
+        ''')
+        unwrapped_exceptions = (NotImplementedError, )
+
+    with assert_raises(NotImplementedError) as e:
+        MyVisitor().parse('bar')
+    assert_in('No visitor method was defined for myrule = ~"[bar]"', str(e.exception))
```

Comment:
I feel like we should delimit this message somehow. Maybe an easy cop-out would be "No visitor method was defined for this expression: blah". Then we don't have to worry about trying to figure out the right kind of quotes to use or introducing some unfamiliar notation like curly braces. What do you think?